conformance/results/contour.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
+date: "2023-07-25T22:11:56Z"
+gatewayAPIVersion: TODO
+implementation:
+  contact:
+  - '@projectcontour/maintainers'
+  organization: projectcontour
+  project: contour
+  url: https://github.com/projectcontour/contour
+  version: "f6c112e60a40ceb9d448354473a8fe7d1a726790"
+kind: ConformanceReport
+profiles:
+- core:
+    result: success
+    statistics:
+      Failed: 0
+      Passed: 28
+      Skipped: 0
+    summary: ""
+  extended:
+    result: success
+    statistics:
+      Failed: 0
+      Passed: 9
+      Skipped: 0
```

Comment:
due to https://github.com/kubernetes-sigs/gateway-api/issues/2206 this is not accurate